kernel/common/binding.rb

```diff
@@ -54,4 +54,12 @@ def eval(expr, filename=nil, lineno=nil)
 
     Kernel.eval(expr, self, filename, lineno)
   end
+
+  def local_variables
+    eval('local_variables')
```

Comment:
@YorickPeterse Indeed, the use of eval is a bit gross. The reason behind it was only to avoid duplicating ```Kernel.local_variables```. Would pushing this logic into ```Rubinius::VariableScope``` be an acceptable option?